SofaKernel/modules/SofaBaseCollision/RigidCapsuleModel.inl

```diff
@@ -107,7 +107,7 @@ void TCapsuleModel<sofa::defaulttype::StdRigidTypes<3,MyReal> >::init()
     _mstate = dynamic_cast< core::behavior::MechanicalState<DataTypes>* > (getContext()->getMechanicalState());
     if (_mstate==NULL)
     {
-        serr<<"TCapsuleModel requires a Rigid Mechanical Model" << sendl;
+        msg_warning()<<"TCapsuleModel requires a Rigid Mechanical Model" << msgendl;
```

Comment:
I think it is a msg_error, as the component init failed and so it is not usable.